pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/plan/pipeline/PipelineBreakerOperator.java

```diff
@@ -50,6 +50,9 @@ public PipelineBreakerOperator(OpChainExecutionContext context,
     _expectedKeySet = ImmutableSet.copyOf(pipelineWorkerMap.keySet());
     _workerEntries = new ArrayDeque<>();
     _workerEntries.addAll(pipelineWorkerMap.entrySet());
+    for (int workerKey : _expectedKeySet) {
+      _resultMap.put(workerKey, new ArrayList<>());
```

Comment:
What's the context for this change?

Reply:
Oh this is b/c we might be null instead of empty pipeline result from a stage. Alternatively we can handle null and empty as equal when retrieve pipeline breaker results